crates/core/src/rpc/method.rs

```diff
@@ -31,11 +31,28 @@ pub enum Method {
 }
 
 impl Method {
-	pub fn parse<S>(s: S) -> Self
+	/// Parse a [Method] from a [str] with any case
+	pub fn parse_case_insensitive<S>(s: S) -> Self
 	where
 		S: AsRef<str>,
 	{
-		match s.as_ref().to_lowercase().as_str() {
+		Self::parse(s.as_ref().to_lowercase().as_str())
```

Comment:
`to_lowercase` is very slow. An chance we can make this case sensitive? If not, have we considered using `to_ascii_lowercase` instead?

Reply:
This was being executed on every WebSocket request. Now it's only done when configuring capabilities when the server starts up. So only done once now.

Reply:
It could be changed to `to_ascii_lowercase` but would that be breaking any behaviour?

Reply:
Ok improved in f82d778 🚀